actions/global_actions.jsx

```diff
@@ -120,7 +120,7 @@ export async function emitPostFocusEvent(postId) {
     if (data) {
         const channelId = data.posts[data.order[0]].channel_id;
         const channel = ChannelStore.getChannelById(channelId);
-        if (!channel || channel.team_id !== TeamStore.getCurrentId()) {
+        if (!channel) {
```

Comment:
The check is unfortunately not redundant. This was put here to fix the case where you have a permalink with the wrong team ID at the beginning. So `/wrong-team/pl/ID`. Without this check, it will break (I forget how). But with it, it will redirect to an error page.

Reply:
You're referring to https://mattermost.atlassian.net/browse/ABC-160, right? This was the issue I had in mind while testing, and I've verified that this continues to fix ABC-160. I couldn't find a case where the channel existed but had a different team -- in all cases, the channel wasn't part of the channel store if you permalink to the wrong team.